runtime/compiler/z/codegen/J9S390PrivateLinkage.cpp

```diff
@@ -2823,6 +2823,73 @@ void TR::J9S390JNILinkage::checkException(TR::Node * callNode,
    generateS390LabelInstruction(self()->cg(), TR::InstOpCode::LABEL, callNode, exceptionRestartLabel);
    }
 
+/**
+ * \brief JNI return value processing:
+ * 1) Unwrap return value for objects or
+ * 2) Enforce a return value of 0 or 1 for boolean return type.
+*/
```

Comment:
Could we Doxygen document this new API and move the documentation to the API definition (header file)?

Reply:
Done.